tests/contrib/langchain/conftest.py

```diff
@@ -147,3 +147,33 @@ def langchain_anthropic(ddtrace_config_langchain, mock_logs, mock_metrics, langc
         yield langchain_anthropic
     except ImportError:
         yield
+
+
+@pytest.fixture
+def langchain_pinecone(ddtrace_config_langchain, mock_logs, mock_metrics, langchain):
+    with override_env(
+        dict(
+            PINECONE_API_KEY=os.getenv("PINECONE_API_KEY", "<not-a-real-key>"),
+        )
+    ):
+        try:
+            import langchain_pinecone
+
+            yield langchain_pinecone
+        except ImportError:
+            yield
+
+
+@pytest.fixture
+def pinecone(ddtrace_config_langchain, mock_logs, mock_metrics, langchain):
+    with override_env(
+        dict(
+            PINECONE_API_KEY=os.getenv("PINECONE_API_KEY", "<not-a-real-key>"),
+        )
+    ):
+        try:
+            import pinecone
+
+            yield pinecone
+        except ImportError:
+            yield
```

Comment:
Why do we need to create a fixture for pinecone? We only follow this fixture formatting for dependencies that we actually patch. Pinecone is not one of them, we can just import it in the test itself.